android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -112,29 +132,45 @@
+    <string name="mozac_feature_prompts_manage_credit_cards_1">Manage cards</string>
     <!-- Text for the title of a save credit card dialog. -->
     <string name="mozac_feature_prompts_save_credit_card_prompt_title">Securely save this card?</string>
     <!-- Text for the title of an update credit card dialog. -->
     <string name="mozac_feature_prompts_update_credit_card_prompt_title">Update card expiration date?</string>
     <!-- Subtitle text displayed under the title of the save credit card dialog. -->
-    <string name="mozac_feature_prompts_save_credit_card_prompt_body">Card number will be encrypted. Security code won’t be saved.</string>
+    <string name="mozac_feature_prompts_save_credit_card_prompt_body" moz:removedIn="123" tools:ignore="UnusedResources">Card number will be encrypted. Security code won’t be saved.</string>
+    <!-- Subtitle text displayed under the title of the save credit card dialog. -->
+    <string name="mozac_feature_prompts_save_credit_card_prompt_body_1">Firefox encrypts your card number. Your security code won’t be saved.</string>
 
     <!-- Address Autofill -->
     <!-- Header for the select address prompt to allow users to fill a form with a saved address. -->
     <string name="mozac_feature_prompts_select_address_2">Select address</string>
     <!-- Content description for expanding the select addresses options in the select address prompt. -->
-    <string name="mozac_feature_prompts_expand_address_content_description">Expand suggested addresses</string>
+    <string name="mozac_feature_prompts_expand_address_content_description" moz:removedIn="123" tools:ignore="UnusedResources">Expand suggested addresses</string>
+    <!-- Content description for expanding the select addresses options in the select address prompt. -->
+    <string name="mozac_feature_prompts_expand_address_content_description_1">Expand saved addresses</string>
     <!-- Content description for collapsing the select address options in the select address prompt. -->
-    <string name="mozac_feature_prompts_collapse_address_content_description">Collapse suggested addresses</string>
+    <string name="mozac_feature_prompts_collapse_address_content_description" moz:removedIn="123" tools:ignore="UnusedResources">Collapse suggested addresses</string>
+    <!-- Content description for collapsing the select address options in the select address prompt. -->
+    <string name="mozac_feature_prompts_collapse_address_content_description_1">Collapse saved addresses</string>
+    <!-- Text for the manage addresses button. -->
+    <string name="mozac_feature_prompts_manage_address" moz:removedIn="123" tools:ignore="UnusedResources">Manage addresses</string>
     <!-- Text for the manage addresses button. -->
-    <string name="mozac_feature_prompts_manage_address">Manage addresses</string>
+    <string name="mozac_feature_prompts_manage_address_1">Manage addresses</string>
```

Comment:
Are these not the same strings? I see it on the spreadsheet, but the string didn't change.

Reply:
Good catch didn't realize. I can ask!